core/assets/jsons/levels/enemySurroundingPlayer.json

```diff
@@ -13,6 +12,7 @@
     3.08,
     12.04
   ],
+  "bgm": "the-doll-thrown-away-in-rain",
```

Comment:
Why this change?

Reply:
No actual reason. Was playing around with the level in level editor and decided it should have music.